jbpm-audit/src/test/java/org/jbpm/process/audit/AbstractAuditLogServiceTest.java

```diff
@@ -365,16 +362,15 @@ public void abortWorkItem(WorkItem workItem, WorkItemManager manager) {
         params.put("s", variableValue);
         long processInstanceId = session.startProcess("com.sample.ruleflow3", params).getId();
         int numActiveProcesses = auditLogService.findActiveProcessInstances().size();
-        assertEquals( "find active processes did not work", initialActiveProcessInstanceSize + 1, numActiveProcesses );
+        // find active processes did not work
+        Assertions.assertThat(numActiveProcesses).isEqualTo(initialActiveProcessInstanceSize + 1);
```

Comment:
You can change this for Assertions.assertThat(numActiveProcesses).isEqualTo(initialActiveProcessInstanceSize + 1).withFailMessage("find active processes did not work"); and delete the comment.

Reply:
OK, thanks for hint!